### ISSUE TITLE
YAML Parser seems unhappy about vault encrypted variables

### ISSUE THREAD

**Issue by tsia** [thedavidwhiteside/ansible-docgen#37]
```
Traceback (most recent call last):
  File "/home/tsiakalt/.local/bin/ansible-docgen", line 7, in <module>
    cli.run()
  File "/home/tsiakalt/.local/lib/python3.7/site-packages/ansibledocgen/cli.py", line 37, in run
    self.dirparser = DirParser(project)
  File "/home/tsiakalt/.local/lib/python3.7/site-packages/ansibledocgen/parser/dir.py", line 20, in __init__
    self.playbookparser.parse_playbooks()
  File "/home/tsiakalt/.local/lib/python3.7/site-packages/ansibledocgen/parser/playbook.py", line 20, in parse_playbooks
    self.parse_playbook(playbook)
  File "/home/tsiakalt/.local/lib/python3.7/site-packages/ansibledocgen/parser/playbook.py", line 52, in parse_playbook
    yamldata = yaml.load(data)
  File "/usr/local/lib/python3.7/dist-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/usr/local/lib/python3.7/dist-packages/yaml/constructor.py", line 51, in get_single_data
    return self.construct_document(node)
  File "/usr/local/lib/python3.7/dist-packages/yaml/constructor.py", line 60, in construct_document
    for dummy in generator:
  File "/usr/local/lib/python3.7/dist-packages/yaml/constructor.py", line 413, in construct_yaml_map
    value = self.construct_mapping(node)
  File "/usr/local/lib/python3.7/dist-packages/yaml/constructor.py", line 218, in construct_mapping
    return super().construct_mapping(node, deep=deep)
  File "/usr/local/lib/python3.7/dist-packages/yaml/constructor.py", line 143, in construct_mapping
    value = self.construct_object(value_node, deep=deep)
  File "/usr/local/lib/python3.7/dist-packages/yaml/constructor.py", line 100, in construct_object
    data = constructor(self, node)
  File "/usr/local/lib/python3.7/dist-packages/yaml/constructor.py", line 429, in construct_undefined
    node.start_mark)
yaml.constructor.ConstructorError: could not determine a constructor for the tag '!vault'
  in "<unicode string>", line 32, column 19:
            password: !vault |
```

**Comment by thedavidwhiteside** [comment]
Resolved in PR #38

**Comment by thedavidwhiteside** [comment]
Will release v0.0.4 soon and will resolve this issue.
